### DISCUSSION TITLE
Emoji support on Qt/Linux

### DISCUSSION THREAD

**Discussion by rgieseke** [orbitalquark/textadept#327] [General]
With the Qt version on Linux Emojis look simpler (and without color) and some seem not to show at all compared to the previous GTK version. Is there a way to configure the fonts used?

**Comment by orbitalquark** [comment]
I know GTK/Pango has fallback fonts for glyphs that do not exist in the 
current font. I don't know if Qt does the same thing though.

**Comment by rgieseke** [comment]
Thanks! Looks like this has been requested for Scintilla as well, but unresolved: https://sourceforge.net/p/scintilla/feature-requests/1456/
